db/changelog/v-1/06-chageset-execution-table.xml

```diff
@@ -22,28 +22,28 @@
                 <constraints nullable="false"/>
             </column>
             <column name="end_time" type="timestamp">
-                <constraints nullable="false" />
+                <constraints nullable="true" />
```

Comment:
@aktsay6 insisted very much, that this field should be not nullable, because we will not read it until execution status changes to `finish` :) But, I guess, since we are making all other fields nullable, it doesn't matter